gui/build.py

```diff
@@ -79,6 +80,20 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    env = dict(os.environ)
+    p = re.search('D:\\\\Python27\\\\Lib', env['PYTHONPATH'])
+    python_path = p.group(0)
+    gui_path = os.path.realpath('.')
+    moz_path = os.path.realpath('..')
+    include_path = '--include-path=%s;%s;%s//site-packages;%s ' % (gui_path, moz_path, python_path, python_path)
+    icon_arg = '--icon=wininst/app_icon.ico '
+    target_arg = '--target-name=mozregression-gui.exe --target-dir=dist'
+    base_name = '--base-name=Win32GUI '
+    arg = icon_arg + base_name + include_path + target_arg
+    call(sys.executable, py_script('cxfreeze'), arg, 'mozregui/main.py')
+    os.system('cd wininst')
+    os.system('makensis wininst.nsi')
```

Comment:
you should make use of call() here too (that calls subprocess) in this way:
call('makensis', 'wininst.nsi', cwd='wininst')

cwd named argument is explained here: https://docs.python.org/2/library/subprocess.html#popen-constructor.